docs/launchpad/quick-start.md

```diff
@@ -78,14 +78,28 @@ Remember that each cluster might have specific setup steps or requirements, espe
 - [x] We have installed all the tooling dependencies on our local machine, which will be used to control the cluster
 - [ ] Next: Copy `sample.helmfile.yaml` to `helmfile.yaml` and edit it to customize and deploy Namespaces on the Kubernetes cluster
```

Comment:
Here maybe we want to convey that with this structure one ought to customize helmfile.yaml to "select" which namespaces to import, and then customize said namespaces to add values for each (in case it's needed)